src/Aspire.Hosting.Azure.CosmosDB/PublicAPI.Shipped.txt

```diff
@@ -8,7 +8,6 @@ Aspire.Hosting.AzureCosmosDBResource.ConnectionStringExpression.get -> Aspire.Ho
 Aspire.Hosting.AzureCosmosDBResource.IsEmulator.get -> bool
 Aspire.Hosting.AzureCosmosExtensions
 override Aspire.Hosting.Azure.AzureCosmosDBEmulatorResource.Annotations.get -> Aspire.Hosting.ApplicationModel.ResourceAnnotationCollection!
-override Aspire.Hosting.Azure.AzureCosmosDBEmulatorResource.Name.get -> string!
```

Comment:
Curious why this changed? Did we end up landing anywhere on how to use these files (or not) going forward?

Reply:
It changed because I removed the override in the class. The override isn't necessary since we pass the Name into the base ctor.